app/controllers/landing_page_controller.rb

```diff
@@ -123,8 +123,14 @@ def preview
     begin
       structure = CLP::LandingPageStore.load_structure(cid, preview_version)
 
-      # Uncomment to use static data instead of dynamic from DB
-      # structure = JSON.parse(CustomLandingPage::ExampleData::DATA_STR)
+      # if static landing page enabled, load data from config/initializers/landing_page.rb
+      if (APP_CONFIG.clp_static_enabled)
+        begin
```

Comment:
don't use exceptions for flow control

Reply:
Changed flow control to use Module#const_defined? in 2451599639851d7721dc02dfc78078e7bdcabdf9